src/app/home/home.component.ts

```diff
@@ -412,17 +474,26 @@ export class HomeComponent implements AfterViewInit {
         this.hasStoppedTypingForEventualMarking = false;
     }
 
-
+    /**
+     * Marks the editor based on the lastly (time-wise) typed character. For example breaking the current line is
+     * considered as a signal to attempt to mark the current written text. Attempting to mark the editor after every
+     * keystroke can annoy the user and will also mean a significantly larger amount of requests made.
+     * @param $event
+     * @private
+     */
     // TODO there's also the paste to be considered
     private shouldMarkEditor($event: KeyboardEvent) {
-        /**
-         * Considers the lastly (time-wise) typed character.
-         */
         const TRIGGERS = ['.', '!', '?', ',', '…', 'Enter', 'Backspace', 'Delete', ' ', ':', ';', '"', '“', '”', '&',
             '(', ')', '/', '\'', '«', '»'];
         return TRIGGERS.includes($event.key);
     }
 
+    /**
+     * No action should be taken in regard to updating aspects of the editor based on some lastly (time-wise) typed
+     * character. For example pressing one of the arrow keys in the keyboard should not alter the editor's state.
+     * @param $event
+     * @private
+     */
     private shouldNotUpdateEditor($event: any) {
```

Comment:
```suggestion
    private shouldNotMarkEditor($event: any) {
```
Not to directly commit this, but shouldMarkEditor seems like it goes in line with shouldNotMarkEditor for consistency

Reply:
True, changed